Song.Tests/SongWriterTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song.Tests
+{
+    public class SongWriterTests
```

Comment:
Global: 

- When testing - prefer to include method under test name. If all tests are testing the same name - you can include it within a class name.
For example: `SongWriteTests_Sing`